services/testTopics/index.js

```diff
@@ -38,9 +38,22 @@ function remove(id) {
     });
 }
 
+function updateTheory(id, theory) {
+    return new Promise((resolve, reject) => {
+        TestTopic.update({ theory }, { where: { id } })
+            .then(result => {
+                TestTopic.findOne({ where: { id } })
+                    .then( topic => resolve(topic))
+                    .catch( err => reject({ status: 500 }));
```

Comment:
it's incorrect, because if we can't find object with id, it's not 500, we have no such record in db simple